locale/hi.yml

```diff
@@ -0,0 +1,1813 @@
+# Files in the locale directory are used for localization.
+# If you want to use locales other than English, follow the instructions in legesher-translations guide.
+# https://legesher.readme.io/docs/translation-guide
+#
+# THANK YOU FOR MAKING LEGESHER LARGER THAN US.
+
+# NOTE: "language-abbreviation"
+language-abbreviation: "hi" 
```

Comment:
Good catch 👍

Reply:
Is it for me?😅

Reply:
So, @madiedgar what should I do now? Should I change this PR?